test/cql-pytest/test_bloom_filter.py

```diff
@@ -0,0 +1,35 @@
+# Copyright 2022-present ScyllaDB
+#
+# SPDX-License-Identifier: AGPL-3.0-or-later
+
+
+# Test inserts `N` rows into table, flushes it 
+# and tries to read `M` non-existing keys.
+# Then bloom filter's false-positive ratio is checked.
+
+import pytest
+import random
+import nodetool
+from util import new_test_table
+
+def test_bloom_filter(cql, test_keyspace):
+    N = 10000 # Number of writes
+    M = 5000  # Number of reads
+    fp_chance = 0.1
+
+    keys = random.sample(range(2147483647), N+M)
+    with new_test_table(cql, test_keyspace, "a int PRIMARY KEY", 
+        f"WITH bloom_filter_fp_chance = {fp_chance}") as table:
+        
+        stmt = cql.prepare(f"INSERT INTO {table} (a) VALUES(?)")
+        for k in keys[:N]:
+            cql.execute(stmt, [k])
+        nodetool.flush(cql, table)
+        
+        for k in keys[N:N+M]:
+            cql.execute(f"SELECT * FROM {table} WHERE a = {k} BYPASS CACHE")
+
+        fp = nodetool.bloom_filter_false_positives(cql, table)
+        ratio = fp / M
+        assert ratio >= fp_chance * 0.8 and ratio <= fp_chance * 1.2
```

Comment:
just curious, have you measured what's the average deviation, or did you just pick (0.8, 1.2) as reasonable enough?